### PR TITLE
 feature/TK-456

### PR DESCRIPTION
**Pull request by ani2fun**
Frontend Select Component to add personal user event

### CODE REVIEW

**Comment by nicmarti** [comment]
We're waiting for PR #332 related to TK-452 before we can merge this PR

File: frontend/src/components/Events/NewEventTemplateForm.js [hunk 1]
```diff
@@ -186,11 +211,12 @@ const NewEventTemplateForm = () => {
                   showTime={{
                     hideDisabledOptions: true
                   }}
-                  format="YYYY-MM-DD HH:mm"
+                  format="DD-MM-YYYY HH:mm"
```
Comment (Ven0mX) [inline frontend/src/components/Events/NewEventTemplateForm.js:214]: Why did you swap the date and year?

**Review by Ven0mX — APPROVED** [review]
LGTM, but I let you a comment

File: frontend/src/components/Events/NewEventTemplateForm.js (hunk 1 above)
Comment (ani2fun) [reply frontend/src/components/Events/NewEventTemplateForm.js:214]: easy to read in front end and follow DD-MM-YYYY convention

**Pull request merged**